### PR TITLE
muhahaha

### PR DESCRIPTION
**Pull request by andre-merzky**
;)

### CODE REVIEW

**Comment by marksantcroos** [comment]
I will only accept this once it aligns perfectly on pixel level for every number of nodes for each of the sets.

**Comment by andre-merzky** [comment]
working on it...

**Comment by andre-merzky** [comment]
hahaha - and now you can't tell if I am kidding, right?? :D

File: src/radical/pilot/agent/radical-pilot-agent-multicore.py [hunk 1]
```diff
@@ -2273,7 +2273,7 @@ def __init__(self, cfg, logger):
 
         if self.agent_nodes:
             self._log.info('Reserved agent node(s): %s' % self.agent_nodes.values())
-            self._log.info('Agent(s) running on node(s): %s' % self.agent_nodes.keys())
+            self._log.info('Utilized agent node(s): %s' % self.agent_nodes.keys())
```
Comment (marksantcroos) [inline src/radical/pilot/agent/radical-pilot-agent-multicore.py]: Too bad the new text is wrong.

**Pull request closed without merging**